pallets/ddc-clusters/src/lib.rs

```diff
@@ -815,6 +816,23 @@ pub mod pallet {
 	}
 
 	impl<T: Config> ClusterManager<T> for Pallet<T> {
+		fn set_last_validated_era(
```

Comment:
I think we need to create a dedicated trait, i.e. `ClusterValidator`, and put the `set_last_validated_era` and other validation-related methods there.

The `ClusterManager` trait is supposed to contain functions related to DDC nodes management
The `ClusterProtocol` trait is supposed to contain functions related to Cluster economics
The `ClusterValidator` trait will contain validation-related functions. 

By keeping those traits separately, you decrease the chance that someone will call your `set_last_validated_era` except the `ddc-payouts` pallet.

It also makes the design clearer and the code more readable.